apps/api/package.json

```diff
@@ -55,6 +55,7 @@
     "swagger-ui-express": "4.2.0",
     "typeorm": "0.2.41",
     "winston": "3.3.3",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.3/xlsx-0.20.3.tgz",
```

Comment:
Any reason why we're pulling xlsx from a cdn instead from an npm repository?

Reply:
yeah, I had the same feeling when I saw this at firs time. It's actually from the official doc.
https://docs.sheetjs.com/docs/getting-started/installation/nodejs

![CleanShot 2024-08-13 at 14 53 41](https://github.com/user-attachments/assets/e5a6e286-67d7-48cb-85ec-9f798f65ce71)

Reply:
Oh, thats very strange. 

All good though